# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchService.cpp
# Commit: 9ee546b9fd2b
# Full Hash: 9ee546b9fd2b093fbb40ff17870fa70df287986f
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2022-02-26 21:36:36
# Regressor Bug: 1754365
# File Overlap Count: 1
# Description:
#   Bug 1754365 - Saving NavigationPreload's PerformanceTimingData into worker's PerformanceStorage. r=dom-worker-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138252
# ==============================================================================

diff -r cf972fe0d961 -r 9ee546b9fd2b dom/fetch/FetchService.cpp
--- a/dom/fetch/FetchService.cpp	Sat Feb 26 08:40:26 2022 +0000
+++ b/dom/fetch/FetchService.cpp	Sat Feb 26 08:40:26 2022 +0000
@@ -185,6 +185,8 @@
       mFetchDriver->GetPerformanceTimingData(initiatorType, entryName));
   MOZ_ASSERT(performanceTiming);
 
+  initiatorType = u"navigation"_ns;
+
   FetchServiceResponse response =
       MakeTuple(std::move(mResponse), performanceTiming->ToIPC(), initiatorType,
                 entryName);